src/controllers/AbvClassificationController.ts

```diff
@@ -0,0 +1,11 @@
+import { Get, JsonController } from 'routing-controllers'
+import { AbvClassification } from '../entity/AbvClassification'
+
+@JsonController('/abvs')
+export class AbvClassificationController {
+  @Get('/')
+  async getAbvClassificationList() {
+    const abvs = await AbvClassification.findAll() 
+    return { abvs }
+  }
+}
```

Comment:
```suggestion
}

```
파일의 맨 마지막엔 공백을 하나 놓아주세오 히히